e2e/cypress/integration/messaging/focus_move_spec.js

```diff
@@ -35,4 +71,24 @@ describe('Messaging', () => {
         cy.get('body').type('B');
         cy.get('#post_textbox').should('be.focused');
     });
+
+    it('M47742 Focus does not move when it has already been set elsewhere', () => {
```

Comment:
I recently corrected the key, sorry about it, please update
```suggestion
    it('M17452 Focus does not move when it has already been set elsewhere', () => {
```